resources/schema.json

```diff
@@ -467,6 +467,52 @@
         }
       }
     },
+    "scopes_old":  {
+      "$id": "#/definitions/scopes_old",
+      "type": "object",
+      "title": "element template scope",
+      "deprecated": true,
+      "description": "@Deprecated - Special scoped bindings that allow you to configure nested elements",
+      "additionalProperties": false,
+      "errorMessage": {
+        "additionalProperties": "invalid scope, object descriptor is only supported for camunda:Connector"
+      },
+      "properties": {
+        "camunda:Connector": {
+          "type": "object",
+          "title": "element template scoped connector binding",
+          "description": "A scoped binding for a camunda connector inside the element",
+          "properties": {
+            "properties": {
+              "$ref": "#/definitions/properties"
+            }
+          }
+        }
+      }
+    },
+    "scopes": {
+      "$id": "#/definitions/scopes",
+      "type": "array",
+      "description": "Special scoped bindings that allow you to configure nested elements",
+      "items": {
+        "$id": "#/scopes/item",
+        "type": "object",
+        "properties": {
+          "type": {
+            "$id": "#scopes/item/type",
+            "type": "string",
+            "pattern": "^(camunda|bpmn):"
```

Comment:
We only allow scopes of type `camunda:Connector` or `bpmn:Error` right? Why not check explicitly for these two?